File: test/resources/hello_virtual/node/pretty.js
var hello = require("./hello");
%INCLUDE_EXTERNAL_PRETTY%

/**
 * New node file
 */
var pretty = function pretty() {
	process.stdout.write("* ");
	hello.hello();
	process.stdout.write(" *\n");
	
	%CALL_EXTERNAL_PRETTY%
};
exports.pretty = pretty;

File: test/resources/hello_virtual/python/pretty.py
'''
Created on 17/07/2013

@author: drodri
'''
import hello
import sys
%INCLUDE_EXTERNAL_PRETTY%


def pretty():
    sys.stdout.write('* ')
    hello.hello()
    sys.stdout.write(' *\n')

    %CALL_EXTERNAL_PRETTY%
